### PR TITLE
suggest change use constraints to none

### PR DESCRIPTION
**Pull request by amsnyder**

### CODE REVIEW

File: in_text/text_data_release.yml [hunk 1]
```diff
@@ -206,7 +206,7 @@ update: none planned
 themekeywords: ["process guided deep learning", "generalization", "explainable AI (XAI)", "stream temperature", "domain shift", "machine learning", "hybrid modeling", "water","temperature","modeling","climate change"]
 
 usage-rules: >-
-  These data are open access usable via creative commons as long as original data providers are acknowledged
+  None
```
Comment (amsnyder) [inline in_text/text_data_release.yml:249]: I think everything we release as the government is under a “public domain” license – while also open, not quite the same as creative commons. When you suggest that original data providers are acknowledged is this a requirement that they be cited for derivative works? Here are some ideas of use constraints that I think might be more appropriate for this release: (1) “None”, (2) “Users are advised to read the associated publication thoroughly to understand appropriate use and data limitations.”, (3) “If using input data for this model, users are advised to check the data source licensing to understand appropriate use.” I think “none” is the response I’ve seen the most in the releases I’ve looked over.

File: in_text/text_data_release.yml [hunk 2]
```diff
@@ -55,95 +55,135 @@ entities:
...
-      attr-label: READMe.md
+      attr-label: River_dl_dataPrep.py
+      attr-def: >-
+        XX
+      attr-defs: This data release # location where attributes are deinfed - can be "This data release" or a link to another data source page
+      data-min: NA
+      data-max: NA
+      data-units: NA
+    -
+      attr-label: ChristianaReaches.csv
```
Comment (amsnyder) [inline in_text/text_data_release.yml:110]: need to add documentation for this file once it gets added

**Review by SimonTopp — APPROVED** [review]
These all look good to me.  Thanks so much for the detailed eye @amsnyder!

File: in_text/text_data_release.yml (hunk 1 above)
Comment (SimonTopp) [reply in_text/text_data_release.yml:249]: Let's stick with "None"

**Pull request merged**